### PR TITLE
Add trash nothing

### PR DESCRIPTION
**Pull request by scyclops**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -637,6 +637,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | Slack | Team Instant Messaging | `OAuth` | Yes | Unknown | [Go!](https://api.slack.com/) |
 | Telegram Bot | Simplified HTTP version of the MTProto API for bots | `OAuth` | Yes | Unknown | [Go!](https://core.telegram.org/bots/api) |
 | Telegram MTProto | Read and write Telegram data | `OAuth` | Yes | Unknown | [Go!](https://core.telegram.org/api#getting-started) |
+| trash nothing! | A freecycling community with thousands of free items posted every day | `OAuth` | Yes | Yes | [Go!](https://trashnothing.com/developer) |
```
Comment (davemachado) [inline README.md]: Please remove the `!` from the title.

**Comment by davemachado** [comment]
And capital letters in the title, please. 😄

**Comment by scyclops** [comment]
Sorry, I was trying to follow the previous entries example which had some lowercase and punctuation brand names (eg. xkcd, Yahoo!, ..) and the canonical brand name for trashnothing.com has always been "trash nothing!" in all lowercase and with an exclamation point.

**Comment by davemachado** [comment]
Sure, no problem. We try to keep the formatting as similar as possible so we don't have a wide divergence of formatting. Thanks!

**Pull request merged**